packages/jest-environment/index.js

```diff
@@ -1,22 +1,41 @@
+const colors = require('colors');
+const importFrom = require('import-from');
+const jestConfig = importFrom.silent(require.resolve('jest'), 'jest-config');
```

Comment:
This should be `NodeEnvironment`, not `jest-config` as we don't need the config here, but the node environment ;)

Reply:
Ah, sh*t, forgot that this ceremony wasn't only about checking the vesion. 😖 I'm a bit unfocused today…